.infra/workers.ts

```diff
@@ -215,4 +215,12 @@ export const workers: Worker[] = [
     topic: 'analytics-api.v1.experiment-allocated',
     subscription: 'api.experiment-allocated',
   },
+  {
+    topic: 'api.v1.source-created',
+    subscription: 'api.source-created-squad',
```

Comment:
This should probably be more specific as to what the subscription actually does.
As in assigning the owner data

Reply:
Agreed